src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json

```diff
@@ -1,8 +1,8 @@
 [
   {
     "id": "Microsoft.Azure.WebJobs.Extensions.Storage",
-    "version": "5.0.0-beta.2",
-    "name": "AzureStorageQueues",
+    "version": "5.0.0-beta.4",
+    "name": "AzureStorage",
     "bindings": [

```

Comment:
@soninaren I am unsure about the importance of the name element here.  Is there any behavior it impacts?

Reply:
No this is just a config file used to generate bundles. Name does not have behavioral impact.